docs/database/database-spans.md

```diff
@@ -478,6 +480,43 @@ Semantic conventions for individual database systems or specialized instrumentat
 MAY specify a different `db.query.summary` format as long as produced summary remains
 relatively short and its cardinality remains low comparing to the `db.query.text`.
 
+## Propagating context to databases
+
+**Status**: [Development][DocumentStatus]
+
+Instrumentations SHOULD propagate the context information to the SQL queries following [sqlcommenter](https://google.github.io/sqlcommenter/spec/).
```

Comment:
I'm concerned about pushing this as a standard in light of https://github.com/google/sqlcommenter/issues/284

Reply:
The plan is to use `sqlcommenter` only with attributes that do not change frequently. In the SQL Server documentation, I have added the section below. @trask Does this plan seem good to you?

> Instrumentations SHOULD make use of [SET CONTEXT_INFO](https://learn.microsoft.com/en-us/sql/t-sql/statements/set-context-info-transact-sql?view=sql-server-ver16) to add high-cardinality information(e.g. `traceparent`) as adding sql comments to queries changes their unique identifier in SQL Server.

Reply:
@lmolkova The `traceparent` format is more widely used, so I prefer adopting that. What are your thoughts?